### DISCUSSION TITLE
Dialog as <dialog>

### DISCUSSION THREAD

**Discussion by guaiamum** [tailwindlabs/headlessui#718] [Ideas]
First of all, love your work! Really streamlines creating websites from scratch! 🚀

So I was trying to render a Dialog component as passed the prop `as="dialog"`, which does not render the content, because the [dialog element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) does not have the attribute `open` it needs.
I'm assuming would be an easy feature to add [in the props we control](https://github.com/tailwindlabs/headlessui/blob/main/packages/%40headlessui-react/src/components/dialog/dialog.tsx#L98)...

What do you think? Any major implications with focus and inner forms handling?

<img width="281" alt="Screenshot 2021-08-04 at 13 39 32" src="https://user-images.githubusercontent.com/21321095/128182123-a32cc4bb-b917-4560-9023-b21371823e70.png">

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

I wouldn't use the `as="dialog"`, similar to how you would not use an `as="select"` on the `Listbox` component.
If you want to (and can) use the native `dialog` html tag, then I would use that one directly without Headless UI.

**Reply by guaiamum** [reply]
Hmmm, got it! Thanks for the reply :)
